views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,5 @@
+<div><img src="{{this.image_url}}" alt=""></div>
+<p>Nombre:{{this.name}}</p>
```

Comment:
Cuando utilizamos this estamos haciendo referencia al mismo objeto, entonces puedes utilizarlo o no.
Yo recomiendo omitirlo para que no te confundas y tu código se vea mas limpio aun. 👍